apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -337,11 +337,13 @@ func authnTypes(schemas *types.Schemas) *types.Schemas {
 					Input:  "setPasswordInput",
 					Output: "user",
 				},
+				"refreshattributes": {},
 			}
 			schema.CollectionActions = map[string]types.Action{
 				"changepassword": {
 					Input: "changePasswordInput",
 				},
+				"refreshattributes": {},
```

Comment:
dont like the name because the word user attribute is currently not exposed in our api. can you switch to
refreshAuthProviderAccess

Reply:
:+1: